src/puppetlabs/trapperkeeper/core.clj

```diff
@@ -32,13 +32,13 @@
   (`init`, `start`, `stop`) as you see fit;  if you'd like to have the trapperkeeper
   framework block the main thread to wait for a shutdown event, call `init`,
   `start`, and then `run-app`."
-  [services config-data]
+  [services config-data-fn]
```

Comment:
This seems like a breaking change. I don't know how important it is in production uses but it breaks the user.clj in the template.